tests/test_beverage_distributor/tests/test_action_server.py

```diff
@@ -0,0 +1,117 @@
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import fields
+from odoo.tests.common import TransactionCase
+from dateutil.relativedelta import relativedelta
```

Comment:
We'll try to follow the guidelines
```suggestion
from dateutil.relativedelta import relativedelta

from odoo import fields
from odoo.tests.common import TransactionCase
```